vispy/visuals/collections/util.py

```diff
@@ -122,7 +122,10 @@ def fetchcode(utype, prefix=""):
     int i = index_x;
     vec4 _uniform;\n"""
 
-    _utype = {name: count for name, count, _ in _utype}
+    orig = _utype
+    _utype = dict()
+    for name, count, _ in orig:
+        _utype[name] = count
```

Comment:
ahh, wasn't sure if there was a Py26-friendly way to do it, but this is probably it